src/main/java/io/antmedia/rest/BroadcastRestService.java

```diff
@@ -827,6 +827,16 @@ public Result stopStreamingV2(@ApiParam(value = "the id of the broadcast.", requ
 	public String[] searchOnvifDevicesV2() {
 		return super.searchOnvifDevices();
 	}
+	
+	@ApiOperation(value = "Get The Profile List for an ONVIF IP Cameras", notes = "Notes here", response = Result.class)
+	@POST
+	@Path("/onvif-device-profiles")
+	@Produces(MediaType.APPLICATION_JSON)
+	public String[] getOnvifDeviceProfiles(@ApiParam(value="Address") @QueryParam("address") String address, 
```

Comment:
Please don't user username, password, etc. They should be already provided. Please use the same way in `moveIPCamera`